2.4/Dockerfile

```diff
@@ -38,22 +38,29 @@ RUN yum install -y epel-release && \
     # Loosen permission bits to avoid problems running container with arbitrary UID
     chmod -R a+rwx /opt/rh/mongodb24/root/var/lib/mongodb
 
-COPY run-mongod.sh /usr/local/bin/
-COPY contrib /var/lib/mongodb/
-
 # Loosen permission bits to avoid problems running container with arbitrary UID
 RUN chmod -R g+rwx /var/lib/mongodb
 
+# Get prefix path and path to scripts rather than hard-code them in scripts
+ENV CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
+    MONGODB_PREFIX=/opt/rh/mongodb24/root/usr \
```

Comment:
@rhcarvalho It was added to provide similar env variables as mysql-container (to make evn "same" in all docker images). Other users does not have it, so I think it could be removed now.